index.html

```diff
@@ -224,6 +224,48 @@ <h3 class="skill-heading">Skills</h3>
                                 </li>
                                 <li class="bi-project6"></li>
                               </ul>
+
+                              <footer>
+                                <section>
+                                    <img src="assets/icons/image_geometry_1.svg" alt="">
+                                    <h4>Get started</h4>
+                                    <p class="get-paragraph">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+                                </section>    
+                                <section class="form-container">
+                                    <form action="https://formspree.io/f/xwkjrzvw" method="post">
+                                        <ul class="section-form">
+                                            <li>
+                                                <label class="label-1">your name</label>
+                                                <input class="name-text" type="text" id="name" name="user-email" placeholder="Thierry..." required maxlength="30">
+                                            </li>
+                                            <li>
+                                                <label class="label-1">your email</label>
+                                                <input class="email-border" type="text" id="email" name="email" placeholder="noumbuht@" required>
+                                            </li>
+                                            <li>
+                                                <label class="label-1">your Message</label>
+                                                <textarea class="textarea1 textarea2" id="message" placeholder="Enter your message here..." required maxlength="500"></textarea>
+                                            </li>
+                                        </ul>
+                                        <div class="start-container"><button class="Start-collaboration"><p>Start collaboration</p></button></div>
```

Comment:
- [ ] I kindly suggest that you add the different button interactions including hover and active (when it's pressed) as shown in the Figma template below:

<img width="279" alt="image" src="https://user-images.githubusercontent.com/52934128/198187731-380b420a-fad9-4bfe-b901-788a7cb9f6b7.png">

the button on the pressed state has the `background: #FFFFFF;`, `color: #1A2236;`, `border-width: 2px` and on hover has the `background: #FFFFFF;`, `color: #1A2236;` `border-width: 1px`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color:  #FFFFFF;;
   color: #1A2236
   border-width: 2px;
}
```
**for hover state**
```
button:hover {
    background-color: #FFFFFF;;
    color: #1A2236
    border-width: 1px;
}